client/packages/programs/src/JsonForms/components/PeriodSearch.tsx

```diff
@@ -29,11 +32,23 @@ const UIComponent = (props: ControlProps) => {
   const programId = options?.findByProgram
     ? extractProperty(core?.data, 'programId')
     : null;
-  const { data, isLoading } = usePeriodList(
+  const today = new Date();
+  const lastDayOfMonth = DateUtils.lastDayOfMonth(today);
+  const { data, isFetching, fetchNextPage } = usePeriodList(
```

Comment:
nit but I like consistency of isLoading which we use everywhere else. Is there a reason to change to fetching?

Reply:
cause we are fetching next page so the loading was just changed to fetching to match, but guess can use isLoading

Reply:
I'm easy, just a minor preference